src/hooks/useKeypress.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useContext } from 'react'
+import StdinContext from '../components/StdinContext'
+
+// TODO: memo? useRef? Will this re-init on render?
+// Should this re-initialize only if the keypressHandler changes?
```

Comment:
shouldn't be necessary for you to think about, just putt everything you pass into the closure in the dep array